cluster/node-pools/worker-splitaz/stack.yaml

```diff
@@ -135,7 +135,7 @@ Resources:
 {{ end }}
   LaunchTemplate:
     Properties:
-      LaunchTemplateName: '{{ $data.Cluster.LocalID }}-{{ $data.NodePool.Name }}'
```

Comment:
Removed `$data` for consistency.

Here `$data` is equivalent to dot `.`. Note also `{{- if eq .Cluster.ConfigItems.kuberuntu_distro_worker "jammy" }}` below that does not use `$data`